src/zenml/integrations/kaniko/image_builders/kaniko_image_builder.py

```diff
@@ -231,6 +233,88 @@ def _run_kaniko_build(
         pod_name: str,
         spec_overrides: Dict[str, Any],
         build_context: "BuildContext",
+    ) -> None:
+        """Runs the kaniko build in kubernetes with provided connectors.
+        Args:
+            pod_name: Name of the Pod that should be created to run the build.
+            spec_overrides: Pod spec override values.
+            build_context: The build context.
+        Raises:
+            RuntimeError: If the process running the Kaniko build failed.
+        """
+        connector = self.get_connector()
+        if connector:
+            logger.info("Using kubernetes connector to run the kaniko build.")
+            api_client = connector.connect()
+
+            if not isinstance(api_client, kubernetes.client.api_client.ApiClient):
+                raise RuntimeError(f"Expected ApiClient, got {type(api_client)}")
+            core_api = kubernetes.client.CoreV1Api(api_client)
+
+            # Second step define the kaniko pod spec
+            container = kubernetes.client.V1Container(
+                name=pod_name,
+                image=self.config.excutor_image,
+                args=[
+                    f"--destination={self.config.target_image}",
+                    "--container=tar://stdin",
```

Comment:
This should be `--context=...`, and it depends on the configuration, it isn't always `stdin`.